test-resources/headless-core-stub/lambdas/callback/src/services/configuration-helper.ts

```diff
@@ -0,0 +1,34 @@
+import { getParametersByName } from "@aws-lambda-powertools/parameters/ssm";
+
+const commonParameterPrefix = process.env.AWS_STACK_NAME || "common-cri-api";
+const testResourcesParameterPrefix = process.env.TEST_RESOURCES_STACK_NAME || "test-resources";
+
+export class ConfigurationHelper {
+    public async getParameters(clientId: string) {
```

Comment:
@mikebeeby has a similar thing in his PR [here](https://github.com/govuk-one-login/ipv-cri-common-lambdas/pull/414/files#diff-2683b3e55be29012b71a316aeed5ee2cd802269e879a1a7ce14b09633d3652db). Don't mind which approach we take but please can we align?

Reply:
I and @mikebeeby  agreed we will end up duplicating stuff and would come back to refactor

Reply:
Nice, is it worth catching up on that before we merge?